dcm4che-core/src/test/java/org/dcm4che3/io/stream/FileURIImageInputStreamSpiTest.java

```diff
@@ -0,0 +1,49 @@
+package org.dcm4che3.io.stream;
+
+import org.junit.Assume;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.*;
+
+public class FileURIImageInputStreamSpiTest {
```

Comment:
File URIs are very difficult to work with, and UNC names are the worst.  These tests are designed to catch issues that result from the FileSystem implementation from rejecting invalid URIs.